src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/Example3.java

```diff
@@ -0,0 +1,43 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="JavadocParagraph"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocparagraph;
+
+// xdoc section -- start
+// violation 6 lines below '<p> tag should not precede HTML block-tag '<h1>''
+// violation 9 lines below '<p> tag should not precede HTML block-tag '<pre>''
+// violation 11 lines below '<p> tag should not precede HTML block-tag '<ul>''
```

Comment:
11 is too much, please make javadoc over methods to make javadoc smaller in size.
we can have 2-3 methods to show what we need.
lets max distance from comment to violation as 7.

Reply:
Done. Divided the javadoc into separate parts.